tests/components/advantage_air/test_climate.py

```diff
@@ -106,8 +126,8 @@ async def test_climate_async_setup_entry(hass, aioclient_mock):
     assert state
     assert state.attributes.get("min_temp") == 16
     assert state.attributes.get("max_temp") == 32
-    assert state.attributes.get("measuredTemp") == 25
-    assert state.attributes.get("setTemp") == 24
+    assert state.attributes.get("temperature") == 24
+    assert state.attributes.get("current_temperature") == 25
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6333224)